qb/test/select.test.ts

```diff
@@ -1192,6 +1192,18 @@ test("filter on link prop in nested path", async () => {
   }));
   await query.run(client);
 });
+
+// EdgeQL limitation
+// test("link prop on backlink", async()=>{
```

Comment:
Just curious - was commenting this out intentional?

Reply:
Yes - this is something that doesn't work currently but I wanted to leave a record for later.